api/types/installers/agentless-installer.sh.tmpl

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+(
+  flock -n 9 || exit 1
+  if grep -q 'TrustedUserCAKeys /etc/ssh/teleport_user_ca.pub' "$SSHD_CONFIG"; then
```

Comment:
nit: should openssh-keys-path be configurable through the script,  just to be consistent? otherwise it could be weird that sshd config is somewhere else but keys are always in `/etc/ssh`